modules/wms/src/services/ogc/wms-service.ts

```diff
@@ -363,6 +363,7 @@ export class WMSSource extends ImageSource<WMSSourceProps> {
     wmsParameters: WMSGetFeatureInfoParameters,
     vendorParameters?: Record<string, unknown>
   ): string {
+    wmsParameters = this._getWMS130Parameters(wmsParameters);
```

Comment:
What is this scoping magic? does `wmsParameters = ` assignment assign to `this.wmsParameters` ?

No JavaScript, that's a bad JavaScript!
![image](https://github.com/visgl/loaders.gl/assets/210009/322ca7ac-80a4-4d20-a37f-647765aeb044)